devel/ansible/roles/dev/files/.bashrc

```diff
@@ -3,4 +3,10 @@
 export FLASK_APP=/vagrant/securitas/app.py
 export SECURITAS_CONFIG_PATH=/home/vagrant/securitas.cfg
 
-alias securitas-devel="poetry run flask run --host=0.0.0.0"
\ No newline at end of file
+alias securitas-start="sudo systemctl start securitas.service 'The Application is running on http://0.0.0.0:5000'"
```

Comment:
What does the last text argument do?

Reply:
I forgot to add echo to each, just providing feedback as systemctl doesn't.